src/setup/playwright.ts

```diff
@@ -18,11 +18,12 @@ export async function launchPlaywright(
     browser = await (
       await import("playwright")
     ).chromium.launchPersistentContext(tmpdir, {
-      headless: false,
+      headless: true,
```

Comment:
We should allow this in options again: https://github.com/ChainSafe/dappeteer/blob/master/src/types.ts#L24